src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -390,6 +399,33 @@ void ViewSceneDialog::assignBackgroundColor()
 }
 
 //------------------------------------------------------------------------------
+void ViewSceneDialog::handleUnselectedItem(const QString& name)
+{
+  unselectedObjectNames_.push_back(name.toStdString());
+  newGeometryValue();
```

Comment:
This may be slow when user clicks a checkbox many times in a row. I'll test that out.